File: cpp/dry.cpp
#include <iostream>

void DoStuff()
{
	std::cout << "Doing stuff!\n";
}

void DoStuffWithThis(char* value)
{
	std::cout << "Doing stuff whit " << value << "\n";
}

int main()
{
	DoStuff();
	DoStuff();
	
	DoStuffWithThis("This");
	DoStuffWithThis("That");
	
	return 0;
}

File: src/rs/dry.rs
fn do_stuff() {
    println!("Doing stuff!");
}

fn do_sum(a :i32, b :i32) -> i32 {
    a + b
}

fn main() {
    do_stuff();
    println!("2 + 1 = {}", do_sum(2, 1));
}
